react-components/src/components/CacheProvider/AssetMappingCacheProvider.tsx

```diff
@@ -38,6 +38,53 @@ const useAssetMappingCache = (): AssetMappingCache => {
   return content.cache;
 };
 
+export const useGenerateNode3DCache = async (
+  cadModelOptions: CadModelOptions[],
+  assetMappings: ModelWithAssetMappings[] | undefined
+): Promise<void> => {
+  const assetMappingCache = useAssetMappingCache();
+
+  useMemo(() => {
```

Comment:
Hooks should not be `async` you cannot `await` inside a React component. So this `useMemo` can then be a `useEffect`

Reply:
https://github.com/cognitedata/reveal/pull/4652/commits/01d9e4192802dde46298958dce3453c03f561cce